src/mediaserver.py

```diff
@@ -48,6 +48,14 @@ def chunked(size, source):
     return Response(chunked(block_size, data), mimetype=mimetype)
 
 
+# for testing
```

Comment:
THis has to be solved differently: First, do not include test code in the main code. Second, you have to test failure cases. What happens if it does not respond, respond wir errors or with invalid data.

Please find a way to mock the "auth-service" (auth.py) in your tests. maybe dependency injection helps.

Reply:
How would dependency injection help here?

Reply:
You can swap implementations at runtime